src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -5570,6 +5570,65 @@ public void archiveVersion(Long id) {
             }
         }
     }
+    
+    public boolean isArchivable() {
+        String className = settingsWrapper.getValueForKey(SettingsServiceBean.Key.ArchiverClassName, null);
```

Comment:
For these booleans, does it make sense to make them nullable Booleans where the logic is only performed once and if it's not null just return the value? (like we do for isHasValidTermsOfAccess)